beacon-chain/core/epoch/epoch_processing.go

```diff
@@ -146,6 +146,10 @@ func ProcessRegistryUpdates(ctx context.Context, state state.BeaconState) (state
 	if churnLimit < limit {
 		limit = churnLimit
 	}
+	// Cap churn limit to max per epoch churn limit. New in EIP7514.
+	if limit > params.BeaconConfig().MaxPerEpochActivationChurnLimit {
```

Comment:
where is the minimum set for this

Reply:
Minimal is used in other places. It's not part of the change besides the value